index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
+language using a right-to-left text layout:
+      </p>
+
+      <pre class="example nohighlight" title="Internationalized Arabic text with a right-to-left layout">
+"statusReason": {
+  "value": "<span class="highlight">إلغاء</span>",
+  "language: "<code>ar</code>-x-d-<code>rtl</code>"
+}
+      </pre>
+
+      <p class="note">
+Utilization of the design pattern above assumes that the
+The JSON-LD <a href="#extensibility">extension</a> associated with the
+<a>property</a> utilizes the <code>@scoped</code> feature of JSON-LD to
+alias the <code>@language</code> and <code>@value</code> keywords to
+<code>language</code> and <code>value</code>, respectively.
+      </p>
+
+    </section>
+
+    <section class="informative">
+      <h3>Multiple Languages</h3>
+
+      <p>
+When multiple languages are utilized in the same value, a more complex
+mechanism is required. [[!JSON-LD]] also supports the expression of language
+information using the <code>rdf:HTML</code> string literal type, as described
+in [[!JSON-LD]],
 <a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
 This mechanism enables developers to use the internationalization features
 present in HTML to express language information. For example, consider the
 following key-value pair:
 
-          <pre>
-"nameHtml": "&lt;span lang="jp">和美&lt;/span>"
+          <pre class="example nohighlight" title="Expressing multiple languages in single text string">
+"nameHtml": "&lt;span <span class="highlight">lang="en"</span>>Kazumi&lt;/span> (&lt;span <span class="highlight">lang="jp"</span>>和美&lt;/span>)"
           </pre>
 
           <p>
 If the JSON-LD Context expressed that any value associated with the
 <code>nameHtml</code> property as being of type <code>rdf:HTML</code>, then
 software agents rendering the property can deterministically identify the
-language in use.
+various languages in use. JSON implementations following the
+<code>@context</code> ordering requirements in this specification will also be
+able to benefit from this feature.
           </p>
         </li>
       </ul>
     </section>
 
     <section class="informative">
-      <h3>Text direction</h3>
-
-      <p>
-When expressing human-readable text content in the data model, it is important
-to be able to explicitly encode the text direction. JSON and its derived formats
-use the UTF-8 encoding of [[Unicode]] for serializing and transmitting text.
-This enables text direction to be directly determined in simple cases,
-but the mechanism is insufficient for general use.
-      </p>
+      <h3>Varied Text Directions</h3>
 
       <p>
-[[!JSON-LD]] supports the expression of text layout information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
-<a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
+When multiple text directions are utilized in the same value, a more
```

Comment:
This section is wrong in several ways.

First, multiple text directions may not require anything additional. The whole point of the Unicode Bidi Algorithm is that, except for certain edge cases, if you have the base direction set correctly, you can determine the display of mixed direction text.

Second, while HTML markup is preferred in a markup context, Unicode provides bidi control characters to handle the aforementioned corner cases, so `is required` is too strong.

Reply:
Rewrote the section, merged it into a "complex markup" section, removed the "is required" language. Fixed in b2f3fdb952a9ca4a96411170bb5d465334cdcad3.